datastream/tools/verify_split.py

```diff
@@ -0,0 +1,14 @@
+import json
+from pathlib import Path
+
+
+def verify_split(old_path, new_path):
```

Comment:
If we use pydantic here you can type the arguments with `Path` and let pydantic convert strings to Path by using `@validate_arguments`